packages/fxa-shared/speed-trap/performance-factory.js

```diff
@@ -2,33 +2,33 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { NAVIGATION_TIMING_FIELDS } from './timing-fields'
+import { NAVIGATION_TIMING_FIELDS } from './timing-fields';
 
 /**
  * Small util for determining if a browser went to sleep.
  */
 class SleepDetection {
-
```

Comment:
The changes on this class were just auto formatting kicking in. Not sure why this didn't get run the first time...